CRM/Contribute/BAO/Contribution.php

```diff
@@ -4684,6 +4694,7 @@ public static function completeOrder(&$input, &$ids, $objects, $transaction, $re
       civicrm_api3('Contribution', 'sendconfirmation', array(
         'id' => $contribution->id,
         'payment_processor_id' => $paymentProcessorId,
+        'is_email_receipt' => $values['is_email_receipt'],
```

Comment:
Looks like `is_email_receipt` wasn't included as part of the `$allowedParams` in `sendconfirmation`. Committed and will be included in the updated PR.